tests/test_e2e.py

```
@@ -16,25 +16,26 @@
+    port = os.environ.get("PORT", "6379")
     target = os.environ.get("TARGET",
                             "target/debug/libfalkordb.dylib" if platform.system() == "Darwin" else "target/debug/libfalkordb.so")
-    r = Redis()
+    r = Redis(port=port)
     try:
         r.ping()
-        client = FalkorDB()
+        client = FalkorDB(port=port)
         g = client.select_graph("test")
         return
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)